src/modules/pos/validator/components/EditStake/index.js

```diff
@@ -17,22 +17,26 @@ import WalletVisual from '@wallet/components/walletVisual';
 import routes from 'src/routes/routes';
 import TokenAmount from '@token/fungible/components/tokenAmount';
 import WarnPunishedValidator from '@pos/validator/components/WarnPunishedValidator';
+import { usePosExpectedSharedRewards } from '@pos/reward/hooks/queries/useStakingRewards';
 import { useLatestBlock } from '@block/hooks/queries/useLatestBlock';
 import { useAuth } from '@auth/hooks/queries';
 import { PrimaryButton, SecondaryButton, WarningButton } from 'src/theme/buttons';
+import RewardDurationSwitch from '@wallet/components/RegisterMultisigForm/CategorySwitch';
```

Comment:
File is called CategorySwitch while we import this as RewardDurationSwitch, unnecessary confusion.

Reply:
I think eniola wanted this be an alias so that its easy to understand what that switch is about, but not a deal breaker we can rename if there is strong case

Reply:
Yes, this is very confusing. I remember we talked about this long time ago that we should only allow named export (not export default, to prevent this behaviour.